db/internal_stats.h

```diff
@@ -699,8 +732,8 @@ class InternalStats {
   // Per-DB stats
   std::atomic<uint64_t> db_stats_[kIntStatsNumMax];
   // Per-ColumnFamily stats
-  uint64_t cf_stats_value_[INTERNAL_CF_STATS_ENUM_MAX];
-  uint64_t cf_stats_count_[INTERNAL_CF_STATS_ENUM_MAX];
+  std::atomic<uint64_t> cf_stats_value_[INTERNAL_CF_STATS_ENUM_MAX];
+  std::atomic<uint64_t> cf_stats_count_[INTERNAL_CF_STATS_ENUM_MAX];
   // Initialize/reference the collector in constructor so that we don't need
```

Comment:
Rather than have the potential for bogus stats, I decided to follow what `db_stats_` does and just use `std::atomic`. Let me know if this is problematic